clients/python/agentic-sandbox-client/README.md

```diff
@@ -1,13 +1,21 @@
 # Agentic Sandbox Client Python
 
-This Python client provides a simple, high-level interface for creating and interacting with sandboxes managed by the Agent Sandbox controller. It's designed to be used as a context manager, ensuring that sandbox resources are properly created and cleaned up.
+This Python client provides a simple, high-level interface for creating and interacting with
+sandboxes managed by the Agent Sandbox controller. It's designed to be used as a context manager,
+ensuring that sandbox resources are properly created and cleaned up.
+
+This client is designed for a scalable architecture where a central Kubernetes Gateway routes
+traffic to a router pod, which then forwards requests to the correct sandbox based on an HTTP header.
 
 ## Usage
 
 ### Prerequisites
 
-- A running Kubernetes cluster (e.g., `kind`).
+- A running Kubernetes cluster (e.g., Kind) with the Gateway API enabled.
```

Comment:
minor nit: probably remove "Kind" since GKE Gateway Class and HealthCheck CRDs do not exist there